Amazon.IonDotnet/Tree/Impl/IonValue.cs

```diff
@@ -228,7 +228,14 @@ public IReadOnlyCollection<SymbolToken> GetTypeAnnotationSymbols()
         /// <param name="annotation">Annotation text.</param>
         public void AddTypeAnnotation(string annotation)
         {
-            AddTypeAnnotation(new SymbolToken(annotation, SymbolToken.UnknownSid));
+            if (annotation == null)
+            {
+                AddTypeAnnotation(new SymbolToken(annotation, 0));
```

Comment:
Just wondering, but do the SymbolToken SIDs have some sort of constants list, because below I see SymbolToken.UnknownSid.

Reply:
SymbolToken has one const field:

`public const int UnknownSid = -1;`